templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete Manufacturer Point</h1>
+  <p>Are you sure you wand to delete the Manufacturer Point: {{ manufacturer }}</p>
+  <p><i> All Cars with this Point will be also deleted! </i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+  </form>
+{% endblock %}
```

Comment:
Code Style issue: Please make sure you use 2 whitespaces indentations in your `.html` files. Currently, it seems like you're using 4 whitespaces.